app/constants/constants.js

```diff
@@ -16,3 +16,7 @@ export const status = {
   REJECTED: 10,
   ON_HOLD: 11
 }
+
+export const MULTIPLE_SPECIES_RELEASE_DATE = new Date('2025-02-26T00:00:00') // 26th Feb 2025
+
+export const DAIRY_FOLLOW_UP_RELEASE_DATE = new Date('2025-01-21T00:00:00') // 21st Jan 2025
```

Comment:
Should these be env vars so they can be adjusted for different environments? No need for the comments I think.

Reply:
We thought about this, because the one in dashboard is an env var. But we decided against it because they have happened and these are the dates they happened on. 

We could move them into env vars so they can be changed, but I'm not sure I see the need for it, as testing is possible without changing these dates.

Reply:
Fair point, they are in the past an will never change.